sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py

```diff
@@ -473,140 +472,3 @@ def get_service_statistics(self, **kwargs: Any) -> MutableMapping[str, Any]:
         kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
         result = self._client.get_service_statistics(**kwargs)
         return result.as_dict()
-
-    @distributed_trace
-    def list_aliases(self, *, select: Optional[List[str]] = None, **kwargs: Any) -> ItemPaged[SearchAlias]:
```

Comment:
The removal of all these alias methods seem to be missing from the CHANGELOG.

Reply:
Added. :)